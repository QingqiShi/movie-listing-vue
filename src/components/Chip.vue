<template functional>
  <div class="chips"><slot /></div>
</template>

<script>
export default {
  props: []
};
</script>

<style lang="scss">
.chips {
  font-family: $font-heading;
  font-size: 0.8rem;
  padding: 0.5rem 1rem;
  border-radius: calc(0.5em + 0.5rem);
  background: $on-primary;
  color: $accent;
  white-space: nowrap;
}
</style>
