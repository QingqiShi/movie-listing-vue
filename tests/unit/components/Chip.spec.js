import { shallowMount } from "@vue/test-utils";
import Chip from "@/components/Chip.vue";

describe("Chip.vue", () => {
  it("renders", () => {
    const wrapper = shallowMount(Chip, { slots: { default: "test" } });
    expect(wrapper).toMatchSnapshot();
  });
});
