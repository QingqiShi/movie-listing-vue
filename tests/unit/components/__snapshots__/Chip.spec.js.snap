// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chip.vue renders 1`] = `<div class="chips">test</div>`;
